### PR TITLE
mas: update to version 1.6.3

### PR DESCRIPTION
**Pull request by kaimingguo**
###### Description

update mas version to 1.6.3

Closes: https://trac.macports.org/ticket/59045

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.15.2 19C57
Xcode 11.3 11C29

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by mf2k** [comment]
Instead of saying "References:" in the commit message, say:

`closes: https://trac.macports.org/ticket/59045
`

**Comment by kaimingguo** [comment]
Sorry, I thought "CLOSE" means closing the ticket on Trac (Because it is not assigned to me, so I use "REFERENCE")

**Comment by mf2k** [comment]
It does not matter who the ticket is assigned to. If the PR fixes the ticket, then it should resolve the ticket via the commit message.

File: sysutils/mas/Portfile [hunk 1]
```diff
@@ -16,9 +16,11 @@ maintainers         {kimuraw @kimuraw} openmaintainer
 platforms           darwin
 
 depends_build       port:carthage
-checksums           md5     ca1560e390fa17988b80248ec7b781b3 \
-                    sha1    e76df7b060cad78554e17ae328e55040d1b2698f \
-                    sha256  50af96b34ade6f0b67a8564e19d2b8818d56186329ce9a5d3f1b0ad3e9df8f6d
+checksums           md5     8222d2f918358434a7cbb3f9c4690d77 \
+                    sha1    ea4381cccc9d8938d444bbb2f456ebbed6f3d47d \
```
Comment (tobypeterson) [inline sysutils/mas/Portfile]: md5/sha1 are obsolete, should be removed

Comment (kaimingguo) [reply sysutils/mas/Portfile]: Okay, I remove MD5/SHA1 checksums.

**Comment by ryandesign** [comment]
> [Travis Build #10112](https://travis-ci.org/macports/macports-ports/builds/641910359) Failed.

It succeeded on 10.14 but failed on 10.13 and earlier because its dependency carthage requires a newer version of the Swift language than the versions of Xcode on those old macOS versions provide. Oh well.

**Pull request merged**